x/ecocredit/migrations/v3/params.go

```diff
@@ -0,0 +1,182 @@
+package v3
+
+import (
+	"regexp"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"github.com/regen-network/regen-ledger/x/ecocredit/core"
+)
+
+var (
+	// This is a value of 20 REGEN
+	DefaultCreditClassFeeTokens = sdk.NewInt(2e7)
+	DefaultBasketCreationFee    = sdk.NewInt(2e7)
+	KeyCreditClassFee           = []byte("CreditClassFee")
+	KeyAllowedClassCreators     = []byte("AllowedClassCreators")
+	KeyAllowlistEnabled         = []byte("AllowlistEnabled")
+	KeyCreditTypes              = []byte("CreditTypes")
+	KeyBasketCreationFee        = []byte("BasketCreationFee")
+)
+
+// TODO: Revisit this once we have proper gas fee framework.
+// Tracking issues https://github.com/cosmos/cosmos-sdk/issues/9054, https://github.com/cosmos/cosmos-sdk/discussions/9072
```

Comment:
I don't think we need to copy over the todo here as this will not be updated when we do have a proper gas fee framework since its specific to the current migration.

```suggestion
```